src/scenes/AcceleratorRouter/ParticipantProjects/EditView/index.tsx

```diff
@@ -40,8 +42,10 @@ const EditView = () => {
   const { phaseVotes } = useVotingData();
   const { goToParticipantProject } = useNavigateTo();
   const { isAllowed } = useUser();
+  const { allApplications } = useApplicationData();
 
   const isAllowedEdit = isAllowed('UPDATE_PARTICIPANT_PROJECT');
+  const isAllowedViewScoreAndVoting = isAllowed('VIEW_PARTICIPANT_PROJECT_SCORING_VOTING');
```

Comment:
This permission is used to gate the fields on the view side, so it seems like we should also be gating on the edit side.

There doesn't currently seem to be an edit equivalent for this permission (`EDIT_PARTICIPANT_PROJECT_SCORING_VOTING`) defined in the ACL, so I used the view permission for now.

Reply:
Can you add one?